install.ps1

```diff
@@ -0,0 +1,26 @@
+$installDirectory = [IO.Path]::Combine($home, ".shorebird")
+
+if (Test-Path $installDirectory) {
+    Write-Output "Shorebird is already installed."
+    return
+}
+
+Write-Output "Installing Shorebird to $installDirectory..."
+
+# & git clone https://github.com/shorebirdtech/shorebird.git -b stable $installDirectory
+& git clone https://github.com/shorebirdtech/shorebird.git -b bo/flutter-batch-files $installDirectory
```

Comment:
This is just for testing I assume and we'll revert before merging?

Reply:
Correct, once https://github.com/shorebirdtech/shorebird/pull/369 lands in stable, I'll uncomment the line above and delete this one.

Reply:
Done, although this won't work until we update `stable`